packages/contracts-bedrock/contracts/universal/SupportedInterfaces.sol

```diff
@@ -4,18 +4,30 @@ pragma solidity ^0.8.0;
 // Import this here to make it available just by importing this file
 import { IERC165 } from "@openzeppelin/contracts/utils/introspection/IERC165.sol";
 
-interface IRemoteToken {
+/**
+ * @title IOptimismMintableERC20
+ * @notice This interface is available on the OptimismMintableERC20 contract. We declare it as a
+ *         separate interface so that it can be used in custom implementations of
+ *         OptimismMintableERC20.
+ */
+interface IOptimismMintableERC20 {
+    function remoteToken() external returns (address);
+
     function mint(address _to, uint256 _amount) external;
 
     function burn(address _from, uint256 _amount) external;
-
-    function remoteToken() external;
 }
 
-interface IL1Token {
+/**
+ * @custom:legacy
+ * @title ILegacyMintableERC20
+ * @notice This interface was available on the legacy L2StandardERC20 contract. It remains available
+ *         on the OptimismMintableERC20 contract for backwards compatibility.
+ */
+interface ILegacyMintableERC20 {
```

Comment:
Hmmm why is `l2Bridge` not part of this interface?

Reply:
We got to this point just by maintaining the previous ERC165 interface derivation: 
https://github.com/ethereum-optimism/optimism/blob/aead527d57858498a9ebfb305bd344f8411698bf/packages/contracts-bedrock/contracts/universal/OptimismMintableERC20.sol#L75-L77

Looking back at the legacy token's actual `supportsInterface` implementation, it doesn't include the `l2Bridge` method either: 

https://github.com/ethereum-optimism/optimism/blob/0c20a11177b3b6d0be74f2c183c58bb198aeadca/packages/contracts/contracts/standards/L2StandardERC20.sol#L35

Reply:
I vote for `remoteToken` to be included in the new interface and no changes to the old interface

Reply:
`remoteToken` was already included, but I've now also added `bridge()` to the new interface.